models/application.py

```diff
@@ -1,14 +1,15 @@
 from ..base_request import BaseRequest
 from ..resources import Message
-from .device import Device
 from ..settings import Settings
+from .config import Config
+import resin.exceptions as ResinException
```

Comment:
Why do you need to alias the import? Just do "from .. import exceptions". Same for the other files.